consensus/istanbul/backend/engine.go

```diff
@@ -367,8 +367,10 @@ func (sb *Backend) VerifySeal(chain consensus.ChainReader, header *types.Header)
 // Prepare initializes the consensus fields of a block header according to the
 // rules of a particular engine. The changes are executed inline.
 func (sb *Backend) Prepare(chain consensus.ChainReader, header *types.Header) error {
-	// unused fields, force to set to empty
-	header.Coinbase = sb.address
+	// If proposer has not set the `Coinbase` via the `tx-fee-recipient` flag, default to the backend address.
+	if header.Coinbase == (common.Address{}) {
```

Comment:
look the other comment

Reply:
let see if we can remove this, we are setting the Coinbase on the `newWork` method already